src/app/_components/StatusBar/IncidentsStatusBar.tsx

```diff
@@ -1,94 +1,59 @@
+import { useColorModeValue } from '@chakra-ui/react';
 import { css } from '@emotion/react';
-import { useEffect, useRef } from 'react';
-import { IncidentImpact } from 'statuspage.io';
+import { useRef } from 'react';
 
 import { useGlobalContext } from '../../../common/context/useGlobalContext';
 import { useUnresolvedIncidents } from '../../../common/queries/useUnresolvedIncidents';
-import { useAppDispatch } from '../../../common/state/hooks';
-import { Flex } from '../../../ui/Flex';
+import { Flex, FlexProps } from '../../../ui/Flex';
 import { Text } from '../../../ui/Text';
 import { TextLink } from '../../../ui/TextLink';
+import { useColorMode } from '../../../ui/hooks/useColorMode';
 import { StatusBarBase } from './StatusBarBase';
-import { setStatusBar, setStatusBarHeight } from './status-bar-slice';
 import { getColor } from './utils';
 
-const incidentImpactSeverity: Record<IncidentImpact, number> = {
-  [IncidentImpact.None]: 0,
-  [IncidentImpact.Minor]: 1,
-  [IncidentImpact.Major]: 2,
-  [IncidentImpact.Critical]: 3,
-};
-
-export function IncidentsStatusBar() {
+export function IncidentsStatusBar(props: FlexProps) {
```

Comment:
I feel like this file should also contain the IncidentsStatusBarWithErrorBoundary, but rename that to IncidentsStatusBar and name this IncidentsStatusBarBase

Reply:
not sure I understood this one, can you check if it's still relevant after the latest push?

Reply:
I see that index.tsx loads this dynamically. So I guess it cannot be helped that IncidentsStatusBarWithErrorBoundary is there and not in this file. I still think it's a better convention to suffix components with "withErrorBoundary". Less wordy and it'll be easier to remember the name and look it up. I think it's better to simplify that components name to  IncidentsStatusBar and make that the file name (not sure why it's index.tsx as there are 2 status bars and it only includes one). Then rename the component in this file to IncidentsStatusBarBase.

Reply:
gotcha, I pushed an update

Reply:
https://github.com/hirosystems/explorer/compare/1e27d5c40aa74b11c532fbeaa49c81393641bd13..2bb7bd0dea30b1bf97e25de4b76dfd7625596f10